app/lib/api_authentication/by_access_token.rb

```diff
@@ -5,7 +5,7 @@ module ByAccessToken
     extend ActiveSupport::Concern
 
     def current_user
-      @current_user ||= authenticated_token&.owner or defined?(super) && super
+      @current_user ||= User.current = authenticated_token&.owner or defined?(super) && super
```

Comment:
This seems to belong to a different PR.

Reply:
What do you mean? that is one of the purposes of the PR. Because the user of the audit is taken from User.current, which used to be `nil` only when the user was identified by access token, for this.

https://github.com/3scale/porta/blob/e6be7e4c686590fc7f537f723ad6567f8b709298/config/initializers/audited_hacks.rb#L180-L185

Reply:
I mean auditing the access token changes AND adding the user to the audits in general are 2 different unrelated changes IMO, even though the Jira asks both in one same ticket.

Reply:
Aha, I see :+1: